cypress-io/cypress
# feat: add angular-ct project setup

- Closes #22818 

### User facing changelog
Detect and scaffold CT project files for Angular applications

### Additional details
Adding a new framework is fairly straightforward as it is all configuration based. Most of the code changes are from added system tests. The [mount](https://github.com/cypress-io/cypress/issues/22813) and [dev-server handler](https://github.com/cypress-io/cypress/issues/22812) PRs need to be merged before this can land.

### Steps to test
- Scaffold a new angular application with `ng new angular-app` (Might need to install ng globally with `npm i -g @angular/cli`)
- `yarn cypress:open` in cypress repo and point it at the new Angular application
- Walk through setup

Won't work until the Angular infra is in which should be soon. You can also run the `packages/launchpad/cypress/e2e/scaffold-component-testing.cy.ts -> angular-cli-unconfigured` test in the launchpad.

### How has the user experience changed?

https://user-images.githubusercontent.com/25158820/180490616-643ab37e-8709-4cd5-9db0-4f615513e196.mov

### PR Tasks
